infrastructure-tests/runTests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -e
+
+# Sanity check
+if [ -z "$PPIPER_INFRA_IT_CF_USERNAME" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_USERNAME is unset"
+    exit 1
+fi
+
+if [ -z "$PPIPER_INFRA_IT_CF_PASSWORD" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_PASSWORD is unset"
+    exit 1
+fi
+
+set -x
+
+# Start a local registry, to which we push the images built in this test, and from which they will be consumed in the test
+docker run -d -p 5000:5000 --restart always --name registry registry:2 || true
+find ../cx-server-companion -type f -exec sed -i "" -e 's/ppiper/localhost:5000\/ppiper/g' {} \;
+
+# Configure our testing Jenkins instance via Configuration as Code (Create build job, register shared libraries, configure executors)
+mkdir -p ../cx-server-companion/life-cycle-scripts/jenkins-configuration
+cp testing-jenkins.yml ../cx-server-companion/life-cycle-scripts/jenkins-configuration
+
+docker build -t localhost:5000/ppiper/container-structure-test:latest ../container-structure-test
```

Comment:
Why do you build it? We do not have tests for it...

Reply:
You also do not use it later on

Reply:
leftover, thanks.